com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassForward.hlsl

```diff
@@ -60,6 +60,9 @@ PackedVaryingsToPS VertTesselation(VaryingsToDS input)
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/TessellationShare.hlsl"
 #endif
 
+#if defined(DEBUG_DISPLAY) && !defined(_DEPTHOFFSET_ON)
+[earlydepthstencil] // quad overshading debug mode write to UAVs
```

Comment:
Do we know if attribute really work with guarded define - i.e arent they always apply even if there is guard ? 
(poking @JulienIgnace-Unity  / @FrancescoC-unity )

Reply:
Gaenerally the preprocessor is run before the attributes are evaluated for compilation (made small example with [branch] here http://shader-playground.timjones.io/82ad51ba693770094363e31f8b3bafe6 you can see in the DXC output how the branch contains the hint or not depending on the define value), so I think this should work fine, though never tried myself with forcing early depth stencil.